mappings/net/minecraft/resource/ResourceMetadata.mapping

```diff
@@ -1,7 +1,18 @@
 CLASS net/minecraft/unmapped/C_bzrwskbr net/minecraft/resource/ResourceMetadata
+	COMMENT Represents a resource metadata.
 	FIELD f_uvkerkbr EMPTY_SUPPLIER Lnet/minecraft/unmapped/C_zccdhhqk;
+		COMMENT Represents a resource supplier of an empty resource metadata.
```

Comment:
```suggestion
		COMMENT Represents a resource supplier of empty resource metadata.
```
Metadata is plural, so "an" doesn't make sense.

Alternatively, it would make sense to say "an empty `ResourceMetadata`" , referring to an instance of the class.

Reply:
```suggestion
		COMMENT Represents a resource supplier of an empty {@code ResourceMetadata}.
```